### ISSUE TITLE
SDK Projects Pack and PackagePath are ignored for Update items. One must Remove and Include for the properties to be honored

### ISSUE THREAD

**Issue by richshadman** [NuGet/Home#9819]
If you have an ItemGroup like the following:
```
<ItemGroup>
	<None Update="somefile.ext">
		<Pack>True</Pack>
		<PackagePath>content\somefolder\</PackagePath>
	</None>
</ItemGroup>
```

It will fail. One must do this for the item to be included:

```
<ItemGroup>
	<None Remove="somefile.ext" />
	<None Include="somefile.ext">
		<Pack>True</Pack>
		<PackagePath>content\somefolder\</PackagePath>
	</None>
</ItemGroup>
```

The Pack (and by extension PackagePath) properties should not be dependent on the Include item type, and should be honored on the Update item as well.

**Comment by zkat** [comment]
Hi! It's kinda hard to suss out exactly what the problem you're experiencing here is. Could you include a minimal repro showing what you're trying to do, and/or possibly a binlog of the operation? More detail about what exactly you're trying to achieve would be super helpful!

**Comment by richshadman** [comment]
The issue was simply that the '<None Update=' item would not get packed, once I changed it to a combination of Remove and Include it would get packed.

I am unable to reproduce it now. I was consistently reproducing it when I submitted it, however now I am seeing the Update items getting packed as well. I recently updated many dependencies on my system including nuget - I am assuming that may be why I cannot reproduce it any more.

I am closing the issue, thank you for checking it.
